plugins/@grouparoo/mongo/__tests__/table-import/import-properties.ts

```diff
@@ -276,6 +288,7 @@ describe("mongo/table/profileProperties", () => {
         });
         expect(values[profile.id]).toEqual([1.42]);
         expect(values[otherProfile.id]).toEqual([0.78]);
+        expect(values[thirdProfile.id]).toBeUndefined;
```

Comment:
Should be a function: `toBeUndefined()`.  There's a few of these

Reply:
😳 Oh my!  Copy/paste fail.  Got those fixed.